generators/generator-defaults.js

```diff
@@ -54,41 +58,41 @@ const appDefaultConfig = {
 
 /** Required config for prompts to be skipped */
 const serverRequiredConfig = {
-    packageName: 'com.mycompany.myapp',
-    cacheProvider: 'ehcache',
-    websocket: false,
-    databaseType: 'sql',
-    prodDatabaseType: 'postgresql',
-    devDatabaseType: 'h2Disk',
-    searchEngine: false,
-    buildTool: 'maven',
+    packageName: defaultNewApplicationOptions[optionNames.PACKAGE_NAME],
+    cacheProvider: defaultNewApplicationOptions[optionNames.CACHE_PROVIDER],
+    websocket: defaultNewApplicationOptions[optionNames.WEBSOCKET],
+    databaseType: defaultNewApplicationOptions[optionNames.DATABASE_TYPE],
+    prodDatabaseType: defaultNewApplicationOptions[optionNames.PROD_DATABASE_TYPE],
+    devDatabaseType: defaultNewApplicationOptions[optionNames.DEV_DATABASE_TYPE],
+    searchEngine: defaultNewApplicationOptions[optionNames.SEARCH_ENGINE],
+    buildTool: defaultNewApplicationOptions[optionNames.BUILD_TOOL],
 };
 
 const serverDefaultConfig = {
     ...serverRequiredConfig,
-    serverPort: 8080,
-    authenticationType: 'jwt',
-    serviceDiscoveryType: false,
+    serverPort: defaultMonolithOptions[optionNames.SERVER_PORT],
+    authenticationType: defaultMonolithOptions[optionNames.AUTHENTICATION_TYPE],
+    serviceDiscoveryType: defaultMonolithOptions[optionNames.SERVICE_DISCOVERY_TYPE],
     enableHibernateCache: true,
 };
 
 /** Required config for prompts to be skipped */
 const clientRequiredConfig = {
-    clientFramework: ANGULAR,
+    clientFramework: defaultMonolithOptions[optionNames.CLIENT_FRAMEWORK],
 };
 
 const clientDefaultConfig = {
     ...clientRequiredConfig,
-    clientTheme: 'none',
-    clientThemeVariant: 'primary',
-    useSass: true,
+    clientTheme: defaultMonolithOptions[optionNames.CLIENT_THEME],
+    clientThemeVariant: defaultMonolithOptions[optionNames.CLIENT_THEME_VARIANT],
+    useSass: defaultMonolithOptions[optionNames.USE_SASS],
 };
 
 const translationDefaultConfig = {
-    enableTranslation: true,
-    nativeLanguage: 'en',
+    enableTranslation: defaultNewApplicationOptions[optionNames.ENABLE_TRANSLATION],
+    nativeLanguage: defaultNewApplicationOptions[optionNames.NATIVE_LANGUAGE],
     get languages() {
-        return [];
+        return defaultNewApplicationOptions[optionNames.LANGUAGES];
```

Comment:
nativeLanguage and languages were removed from jdl defaults.
should be reverted to 
```
nativeLanguage: 'en',
get languages() {
    return ['en', 'fr'];
}
```

Reply:
@mshima : is it the reason why the current main build failed ?

Reply:
I think so. I am not in front a computer now to confirm.

Reply:
Shouldn't we update jdl defaults instead?

Reply:
@Tcharl : feel free to provide another PR, I already merged this fix to the main branch

Reply:
@Tcharl nativeLanguage and languages were dropped from jdl defaults at https://github.com/jhipster/generator-jhipster/pull/12810.
So both are undefined when executing the generator, in those cases we can somehow define nativeLanguage and languages based on nativeLanguage.
See https://github.com/jhipster/generator-jhipster/issues/12900

Reply:
Thanks @pascalgrimaud and @mshima to have fixed it in a blink of an eye: I'm always impressed by the Jhipster's reactivity :-)

Reply:
it was important, there are a lot of PR today :)